fmriprep/workflows/bold/confounds.py

```diff
@@ -492,11 +492,11 @@ def init_carpetplot_wf(mem_gb, metadata, cifti_output, name="bold_carpet_wf"):
 
     # Warp segmentation into EPI space
     resample_parc = pe.Node(ApplyTransforms(
-        float=True,
+        default_value=0,
         input_image=str(get_template(
             'MNI152NLin2009cAsym', resolution=1, desc='carpet',
             suffix='dseg', extension=['.nii', '.nii.gz'])),
-        dimension=3, default_value=0, interpolation='MultiLabel'),
```

Comment:
will this ever not be `dimension=3`?

Reply:
nope, but `dimension=3` is the default, so no need to be set (same for `default_value`)

Reply:
https://github.com/nipy/nipype/blob/4c48a185e4d190071f78a6766ca34c71ff61e10f/nipype/interfaces/ants/resampling.py#L307-L317

from that help string it seems it is inferred if not provided, I think leaving it would be best